src/common/CMakeLists.txt

```diff
@@ -63,7 +63,8 @@ if (BUILD_PYTHON)
   INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/pysirf.py" DESTINATION "${PYTHON_DEST}/sirf")
   file(GLOB PythonFiles "${CMAKE_CURRENT_LIST_DIR}/*.py")
   INSTALL(FILES ${PythonFiles} DESTINATION "${PYTHON_DEST}/sirf")
-
+  # add tests directory
+  add_subdirectory("tests")
```

Comment:
is it really correct to have this in `common`? Seems unlikely as it needs STIR stuff.

Reply:
Because I test with a PET image?

You are right. Is it possible to create a generic `DataContainer`?

I suppose I need to add the same tests for MR images.

Reply:
and for NiftyImageData?